libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -711,33 +709,9 @@ - (BOOL) webView:(UIWebView *) webView shouldStartLoadWithRequest:(NSURLRequest
             [self.commandQueue fetchCommandsFromJs];
             [self.commandQueue executePending];
             return NO;
-        }
-
-        /*
-         * Give plugins the chance to handle the URL.
-         */
-        BOOL anyPluginsResponded = NO;
-        BOOL shouldAllowRequest = NO;
-        for (NSString* pluginName in self.pluginObjects) {
-            CDVPlugin* plugin = [self.pluginObjects objectForKey:pluginName];
-            SEL selector = NSSelectorFromString(@"shouldOverrideLoadWithRequest:navigationType:");
-            if ([plugin respondsToSelector:selector]) {
-                anyPluginsResponded = YES;
-                shouldAllowRequest = (((BOOL (*)(id, SEL, id, int)) objc_msgSend)(plugin, selector, request, navigationType));
-                if (!shouldAllowRequest) {
-                    break;
-                }
-            }
-        }
-        if (anyPluginsResponded) {
-            return shouldAllowRequest;
-        }
-        if ([url isFileURL]) {
-            return YES;
         } else {
             [[NSNotificationCenter defaultCenter] postNotification:[NSNotification notificationWithName:CDVPluginHandleOpenURLNotification object:url]];
         }
-        return NO;
```

Comment:
`UIWebView` was broken for hybrid remote because of this piece of code. The default if everything falls through should `return YES` to load the page. I also removed the obsolete code above and lined things up with the `WKWebView` implementation. The `Cordova` library doesn't require this code to delegate explicitly anymore. Verified that everything's working as expected in `UIWebView` now, both first time load and session refresh flow.